boa/src/value/tests.rs

```diff
@@ -757,6 +757,8 @@ mod cyclic_conversions {
 }
 
 mod abstract_relational_comparison {
+    #![allow(clippy::bool_assert_comparison)]
```

Comment:
Why are we allowing this here?

Reply:
It's because the macro we use uses `assert_eq($expr, $expected)` and when it's replaced it gives, and expected can be `false` or `true` depending on macro invocation, but clippy does not know this and says it should be `assert!($expr)`. And this is not correct for when we call with false since it has to be negated (assert!(!$expr))